crates/biome_plugin_loader/src/snapshots/load_plugin_with_wrong_version.snap

```diff
@@ -0,0 +1,10 @@
+---
+source: crates/biome_plugin_loader/src/lib.rs
+expression: content
+---
+plugin ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  × Cannot load plugin manifest
+    
+    Caused by:
+      version must be 1
```

Comment:
I think these snapshots could be improved by providing the input/files provided.